### PR TITLE
UX and DX: create zkApp CLI contributing.md and use absolute links in README so npm works

### PR DESCRIPTION
**Pull request by barriebyron**
This PR does a few nifty things to improve UX and DX and encourage community contributions:

- creates a contributing file so we can link to it from the What's New in SnarkyJS blog 
- updates README with editorial grooming and adds absolute links that will work on npm (a few busted links on https://www.npmjs.com/package/zkapp-cli, including an examples directory that is someone else's from 9 years ago, oof)
- updates the package.json to help us be more discoverable! 

@ymekuria  it would be cool to include some badges here, like we did for SnarkyJS https://www.npmjs.com/package/snarkyjs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,99 +106,53 @@ npm run build && node ./build/src/index.js
...
-git push origin <your-branch>
-# Submit a pull request
-# To switch back to the released version, run `npm i -g zkapp-cli`
-```
+Note: When you deploy to an alias for the first time, you are prompted to choose which smart contract you want to deploy from those that exist as _named_ exports in your project. The name of the smart contract that you choose is remembered by being saved into your `config.json` for this alias. For safety, the next time you run `zk deploy <alias>` it automatically deploys this _same_ smart contract in the future when deploying to this alias.
 
 ## License
 
-[Apache-2.0](LICENSE)
+[Apache-2.0](https://github.com/o1-labs/zkapp-cli/blob/main/LICENSE)
```
Comment (barriebyron) [inline README.md:158]: the README is scraped to create https://www.npmjs.com/package/zkapp-cli 

I learned that README links must be absolute to work on npm

File: README.md [hunk 2]
```diff
@@ -1,46 +1,62 @@
...
-## Dependencies
+To get started, read through this README file and the Mina Protocol [zkApp Developer](https://docs.minaprotocol.com/zkapps) docs. 
 
-You'll need the following installed to use the zkApp CLI:
+Build foundational knowledge by completing the guided steps in the [zkApp Developer Tutorials](https://docs.minaprotocol.com/zkapps/tutorials).
 
-- NodeJS 16+ (or 14 using `--experimental-wasm-threads`)
-- NPM 6+
-- Git 2+
+## Get Involved
```
Comment (shimkiv) [inline README.md:13]: It will be goo to mention the CONTRIBUTING.md link in this section with brief description.
Other than that looks good, thank you.

**Review by shimkiv — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -104,99 +106,53 @@ npm run build && node ./build/src/index.js
...
-`https://proxy.berkeley.minaexplorer.com/graphql`), and 3.) the transaction fee
-to be used during deployment (in MINA; e.g. `0.01`). The URL is significant
-because this determines which network you're deploying to (e.g. `QANet`,
-`Testnet`, etc).
-
-Note: If your project contains more than one smart contract that you will be
-deploying, we recommend following an alias naming convention such as `testnet-foo`
-and `testnet-bar`, where `testnet` is the name of the network and `foo` and `bar`
-are the name of your smart contracts. Then you can deploy using `zk deploy testnet-foo` and `zk deploy testnet-bar`.
+This interactive command is interactive prompts you to specify a name (can be anything), URL to deploy to, and fee (in
```
Comment (ymekuria) [inline README.md:134]: This sentence reads a little weird to me.

Suggestion:

"this interactive command is interactive prompts you" -> this interactive command prompts you"

Comment (ymekuria) [reply README.md:134]: We should also mention the fee payer config item recently added.

**Pull request merged**